src/stories/ContactInfo.stories.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { withKnobs, boolean, text } from '@storybook/addon-knobs';
+import { ContactInfo } from '../components/ContactInfo';
+
+export default {
+  title: 'ContactInfo',
+  component: ContactInfo,
+  decorators: [withKnobs]
+};
+
+// Data
+const socialData = [
```

Comment:
please remove all unnecessary keys from social and contact mocked data objects, things like (`_id` .. etc)